.github/workflows/_runner-determinator.yml

```diff
@@ -98,26 +107,28 @@ jobs:
               return branch.split("/")[0] in {"main", "nightly", "release", "landchecks"}
 
 
-          def get_workflow_type(issue: Issue, username: str) -> Tuple[str, str]:
-              formatted_username = (
-                  f"@{username}"  # Add the @ symbol to match the format in the issue body
-              )
-
+          def get_workflow_type(issue: Issue, username_check: Iterable[str]) -> Tuple[str, str]:
               try:
-                  user_list = issue.get_comments()[0].body.split()
+                  first_comment = issue.get_comments()[0].body.strip("\n\t ")
 
-                  if user_list[0] == "!":
+                  if first_comment[0] == "!":
```

Comment:
note that this is a behavior change. Making the first line of the issue "!" or "*" while leaving everyone else specified below will no longer opt everyone out/in to the new runners (not that I'm a fan of this particular notation either ways)

Reply:
it will...

the only difference is that, if there is a space in this line, or extra empty lines above it will still work...